src/main/java/uk/gov/hmcts/ccd/v2/internal/resource/UISearchInputsResource.java

```diff
@@ -24,7 +24,7 @@
 
     @Data
     @NoArgsConstructor
-    public class UISearchInput {
+    public static class UISearchInput {
```

Comment:
Required by jackson in tests for inner classes. Not sure how spring boot does internally